### ISSUE TITLE
Error: Not Logged In (Trade Response {})

### ISSUE THREAD

**Issue by smitteh1** [pepzwee/node-csgo-web-tradebot#159]
### Expected behavior
Send trade request

### Actual behavior
Responds with {}
Outputs a number in console i.e  [ '15218821824' ]

### Steps to reproduce the behavior
Setup script bare minimum, 
(Bot info and API Tokens etc)
### Other
1. [Operating System]: Windows 10
2. [Node.js version]: v10.13.0
3. [How did you start the script]: node index.js

After come debugging I suspect its something to do with offer.getUserDetails (160 in /index.js)
I inserted a console.log(detailsError) inside of it
OutPut : Error: Not Logged In

**Comment by iDefineHD** [comment]
The console output is an array with the assetid i presume, so that seems fine i presume. Not logged in could be a problem with the Manager, could you supply the part of the code you are checking, thanks.

**Comment by smitteh1** [comment]
`                    offer.getUserDetails((detailsError, me, them) => {
                        if (detailsError) {
							console.log('Detail Error : ' + detailsError)
                            socket.emit('offer status', {
                                error: detailsError,
                                status: false,
                            })
                        } else if (me.escrowDays + them.escrowDays > 0) {
                            socket.emit('offer status', {
                                error: 'You must have 2FA enabled, we do not accept trades that go into Escrow.',
                                status: false,
                            })
                        } else {
                            offer.send((errSend, status) => {
                                if (errSend) {
                                    socket.emit('offer status', {
                                        error: errSend,
                                        status: false,
                                    })
                                } else {
                                    console.log('[!!!!!] Sent a trade: ', data)
                                    if (status === 'pending') {
                                        socket.emit('offer status', {
                                            error: null,
                                            status: 2,
                                        })
                                        Trade.botConfirmation(data.bot_id, offer.id, (errConfirm) => {
                                            if (!errConfirm) {
                                                socket.emit('offer status', {
                                                    error: null,
                                                    status: 3,
                                                    offer: offer.id,
                                                })
                                            } else {
                                                socket.emit('offer status', {
                                                    error: errConfirm,
                                                    status: false,
                                                })
                                            }
                                        })
                                    } else {
                                        socket.emit('offer status', {
                                            error: null,
                                            status: 3,
                                            offer: offer.id,
                                        })
                                    }
                                }
                            })
                        }
                    })`
yes I think it is the trade manager i'll see if it is outdated

**Comment by smitteh1** [comment]
```                    offer.getUserDetails((detailsError, me, them) => {
                        if (detailsError) {
							console.log('Detail Error : ' + detailsError)
                            socket.emit('offer status', {
                                error: detailsError,
                                status: false,
                            })
                        } else if (me.escrowDays + them.escrowDays > 0) {
                            socket.emit('offer status', {
                                error: 'You must have 2FA enabled, we do not accept trades that go into Escrow.',
                                status: false,
                            })
                        } else {
                            offer.send((errSend, status) => {
                                if (errSend) {
                                    socket.emit('offer status', {
                                        error: errSend,
                                        status: false,
                                    })
                                } else {
                                    console.log('[!!!!!] Sent a trade: ', data)
                                    if (status === 'pending') {
                                        socket.emit('offer status', {
                                            error: null,
                                            status: 2,
                                        })
                                        Trade.botConfirmation(data.bot_id, offer.id, (errConfirm) => {
                                            if (!errConfirm) {
                                                socket.emit('offer status', {
                                                    error: null,
                                                    status: 3,
                                                    offer: offer.id,
                                                })
                                            } else {
                                                socket.emit('offer status', {
                                                    error: errConfirm,
                                                    status: false,
                                                })
                                            }
                                        })
                                    } else {
                                        socket.emit('offer status', {
                                            error: null,
                                            status: 3,
                                            offer: offer.id,
                                        })
                                    }
                                }
                            })
                        }
                    })```

**Comment by smitteh1** [comment]
did `npm install steam-tradeoffer-manager`
same error

**Comment by Rakeo** [comment]
You must get the steam API key from the bot account not your main steam account. So when getting api key be logged into the bot account.

**Comment by smitteh1** [comment]
> You must get the steam API key from the bot account not your main steam account. So when getting api key be logged into the bot account.

this is not the case as now it happens only after the bot relogs
when I restart the application it works perfect but after it restarts the bot it breaks

**Comment by smitteh1** [comment]
now getting same problem as issue #158

**Comment by smitteh1** [comment]
Resolved in #158
